src/main/java/com/google/api/codegen/configgen/transformer/DiscoConfigTransformer.java

```diff
@@ -108,34 +113,61 @@ private LicenseView generateLicense() {
 
   /**
    * Examines all of the resource paths used by the methods, and returns a map from each unique
-   * resource paths to a short name used by the collection configuration.
+   * resource paths to a short name used by the collection configuration. Each resource path is
+   * merely a string describing the fields in the entity, and the resource path may not be the same
```

Comment:
s/may not/might not/

("may not" means it's not allowed - based on context, it doesn't seem like that's what you want)

Reply:
yep, done.